WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -161,6 +161,7 @@ private extension OrderPaymentSection {
         VStack(alignment: .leading, spacing: Constants.taxesSectionVerticalSpacing) {
             taxSectionTitle
             taxLines
+            shippingTax
```

Comment:
What do you think if, instead of adding the `viewModel.shouldShowShippingTax` inside the `@ViewBuilder var shippingTax`, we add the check to the component rendering here instead? eg:
```suggestion
            shippingTax
            .renderedif(viewModel.shouldShowShippingTax)
```
This way the ViewBuilder is only responsible of creating the view, not about deciding if has to be created.